x-pack/plugins/monitoring/public/components/elasticsearch/ccr/ccr.test.js

```diff
@@ -5,7 +5,7 @@
  */
 
 import React from 'react';
-import { shallow } from 'enzyme';
+import { shallowWithIntl } from 'test_utils/enzyme_helpers';
```

Comment:
Where can I find more information about this library? I don't see it here: https://github.com/elastic/kibana/blob/master/x-pack/test_utils/enzyme_helpers.js